build/codegen/mod.rs

```diff
@@ -9,7 +9,7 @@ pub mod stream;
 pub mod structs;
 
 #[derive(Clone)]
-pub struct UniqueIds(Cell<u32>, RefCell<HashMap<String, u32>>);
+pub struct UniqueIds(#[allow(dead_code)] Cell<u32>, RefCell<HashMap<String, u32>>);
```

Comment:
I think if we're going to include this change, it might as well remove the dead field entirely. I think this is just dead code from a past version of the struct, anyways.

Reply:
OK. Let me do it then.